### PR TITLE
Updated grammar of README

### PR DESCRIPTION
**Pull request by benhayehudi**
Hi,

I took the liberty of updating the README file by adding some conjunction words to a few of the sentences to help make it read a bit more clearly. I also separated the word "checkout" into two words "check out," which I believe is the more proper usage here, with its verb form.

Thanks so much,

Ben

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
 # React-Bootstrap Documentation Website
 
-This website is single page app built on
+This website is a single page app built on
 [React](http://facebook.github.io/react/), with styles and structure taken from
 the [Bootstrap](http://getbootstrap.com/) docs website.  The app is statically
-generated to HTML via node and then hosted it by pushing HTML to [GitHub
+generated to HTML via node and then hosted by pushing HTML to [GitHub
```
Comment (taion) [inline docs/README.md]: capitalize "Node"?

Comment (benhayehudi) [reply docs/README.md]: Yes, I believe so. According to the website it is spelled "Node.js". Node, being shorthand should probably still conform to the uppercase proper name style choice. Good call!

**Comment by taion** [comment]
Thanks!

**Comment by benhayehudi** [comment]
My pleasure! Thanks for letting me contribute. Looking forward to getting more involved.

**Pull request merged**